external/webserver/http_client.c

```diff
@@ -1059,12 +1059,48 @@ int http_send_response_chunk(struct http_client_t *client, int status, const cha
 	return HTTP_OK;
 }
 
+static int http_send_buffer(struct http_client_t *client, const char *buf, int len)
+{
+	int  ret = 0;
+	int sndlen = 0;
+	int buflen = 0;
+
+	if (client == NULL) {
+		HTTP_LOGE("client is NULL \n");
+		return -1;
+	}
+
+	sndlen = len;
+	while (sndlen > 0) {
+#ifdef CONFIG_NET_SECURITY_TLS
+		if (client->server->tls_init) {
+			ret = mbedtls_ssl_write(&(client->tls_ssl), (unsigned char *)buf + buflen, sndlen);
+		} else
+#endif
+		{
+			ret = send(client->client_fd, buf + buflen, sndlen, 0);
+		}
+
+		if (ret < 1) {
```

Comment:
Maybe `ret` is successfully sent bytes.
How about renaming `ret` like `send_bytes`or something much better to understand?

Reply:
done